encodings/dict/src/display.rs

```diff
@@ -20,7 +21,10 @@ mod test {
         .into_array();
 
         assert_eq!(
-            x.display().to_string(),
+            x.display_as(DisplayOptions::CommaSeparatedScalars {
+                space_after_comma: true
```

Comment:
I'm afraid Dan is in the minority here. This should be the default

Reply:
What do you mean? That the value display should be the default display?